tests/parse/test_parse_case.py

```diff
@@ -261,3 +262,20 @@ def test_wrong_relations():
     # THEN a PedigreeError should be raised
     with pytest.raises(PedigreeError):
         parse_individuals(samples)
+
+
+
+
+def test_removeNoneValues():
+    d = {"a":"1", "b":2, "c":3}
```

Comment:
Could you follow the GIVEN, WHEN, THEN like in most other tests? I know it's a bit cumbersome for small tests like these but it is nice if the code follows the same patterns. Also it makes the tests really easy to understand

Reply:
Done! In commit "fb3f0695".